src/rpm/triggers.systemd.in

```diff
@@ -115,25 +100,13 @@ end
 -- This script will process files installed in {{TMPFILES_DIR}} to create
 -- tmpfiles automatically. The priority is set such that it will run
 -- after the sysusers file trigger, but before any other triggers.
-if posix.access("/run/systemd/system") then
-    pid = posix.fork()
-    if pid == 0 then
-        assert(posix.execp("systemd-tmpfiles", "--create"))
-    elseif pid > 0 then
-        posix.wait(pid)
-    end
-end
+assert(rpm.execute("systemd-tmpfiles", "--create"))
 
 %transfiletriggerin -P 1000600 udev -p <lua> -- {{UDEV_RULES_DIR}}
 -- This script will automatically update udev with new rules if files
 -- have been installed or updated in {{UDEV_RULES_DIR}}.
-if posix.access("/run/systemd/system") then
-    pid = posix.fork()
-    if pid == 0 then
-        assert(posix.execp("udevadm", "control", "--reload"))
-    elseif pid > 0 then
-        posix.wait(pid)
-    end
+if posix.access("/run/udev/control") then
```

Comment:
Please don't bury unrelated changes like this in refactor commits. I think this is reasonable, but it needs to be a separate commit.

Reply:
Well i'd like to aign with shell version of the triggers at one hit
https://github.com/systemd/systemd/blob/01ae74c8c7f364dd3a1d71a43af00c34f531d3f7/src/rpm/triggers.systemd.sh.in#L78

Reply:
That is fine. But it still needs to be a separate commit.

Reply:
Sure, that is no problem for me.